src/test/java/tests/FilterEvalTests.java

```diff
@@ -37,6 +44,17 @@ public boolean matches(String filterExpr, TurboIssue issue) {
         return Qualifier.process(empty, Parser.parse(filterExpr), issue);
     }
 
+    /**
+     * Helper method for testing SemanticException thrown by different qualifiers
+     */
```

Comment:
Umm... not a very good description of the method. Write it is the form of `Does abc`. e.g. `Confirms that the the given input causes a SemanticException with the correct error message. Causes a test failure if it doesn't.`